### PR TITLE
ACCA_IfcTunnel_Sprint_1_2

### PR DESCRIPTION
**Pull request by michelangelo-acca**
## File

The IFC files produced from ACCA software for the Sprint 1_2.

It contains:

- Project
- UM
- Context
- An empty Group
- Project Classification
- Geolocalization Information

## Covered Concept Templates - Usages Included

*Please list all Usages you have implemented.*

The covered concept templates are:

1. Project Unit Assignment
2. Project Representation Context
3. Project Declaration
4. Project Classification Information
5. Spatial Decomposition

### CODE REVIEW

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [Alignment-basedReferenceView]'), '2;1');
+FILE_NAME ('ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc', '2023-02-16T14:56:46', ('Alfonso D\X2\0027\X0\Urso', 'alfonso.durso@acca.it'), ('ACCA software S.p.A.'), 'ACCA_IFC.dll Version 500', 'ACCA usBIM.infra', $);
+FILE_SCHEMA (('IFC4X4'));
```
Comment (pjanck) [inline files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc:5]: We encourage you to use schema identifier with a hash attached, see [FAQ](https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/blob/main/FAQ.md). (Not necessary to correct now, but the checker will be updated to reject files with wrong schema identifier.)

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#20 = IFCDIRECTION((0., 0., 1.));
+#21 = IFCDIRECTION((1., 0., 0.));
+#22 = IFCAXIS2PLACEMENT3D(#15, #20, #21);
+#23 = IFCRELAGGREGATES('0_td4AF1j5lweAb2kvPx$x', #1, 'Project Container', 'Project to Spatial Element', #7, (#18));
+#24 = IFCCLASSIFICATION('Default Source', 'Default Edition', '2000-01-01', 'Default Name', $, $, $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('2sTjfww6T4Y8G$LfWsT52u', #1, $, $, (#7), #24);
+#26 = IFCGROUP('1Xwt4d0Yn11B_91a8ResIh', #1, 'A group', $, $);
+#27 = IFCRELDECLARES('3R0mWNDGXDnB8Ljwb2_SSu', #1, 'Project Declaration Relationship', $, #7, (#26));
+#28 = IFCMAPCONVERSION(#13, #29, 0., 0., 0., 0., 0., 1.);
+#29 = IFCPROJECTEDCRS('EPSG:3065', 'Istituto Geografico Militare 1995 (IGM95)', 'EPSG:3065', $, 'UTM', '33N', $);
```
Comment (pjanck) [inline files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc:36]: Double checking: is the height-related CRS unknown?

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#19 = IFCLOCALPLACEMENT($, #22);
+#20 = IFCDIRECTION((0., 0., 1.));
+#21 = IFCDIRECTION((1., 0., 0.));
+#22 = IFCAXIS2PLACEMENT3D(#15, #20, #21);
+#23 = IFCRELAGGREGATES('0_td4AF1j5lweAb2kvPx$x', #1, 'Project Container', 'Project to Spatial Element', #7, (#18));
+#24 = IFCCLASSIFICATION('Default Source', 'Default Edition', '2000-01-01', 'Default Name', $, $, $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('2sTjfww6T4Y8G$LfWsT52u', #1, $, $, (#7), #24);
+#26 = IFCGROUP('1Xwt4d0Yn11B_91a8ResIh', #1, 'A group', $, $);
+#27 = IFCRELDECLARES('3R0mWNDGXDnB8Ljwb2_SSu', #1, 'Project Declaration Relationship', $, #7, (#26));
+#28 = IFCMAPCONVERSION(#13, #29, 0., 0., 0., 0., 0., 1.);
```
Comment (pjanck) [inline files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc:35]: Both `XAxisAbscissa` and `XAxisOrdinate` cannot be zero.

@larswik @SergejMuhic Should we introduce a `WHERE` rule here?

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_B.ifc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#20 = IFCDIRECTION((0., 0., 1.));
+#21 = IFCDIRECTION((1., 0., 0.));
+#22 = IFCAXIS2PLACEMENT3D(#15, #20, #21);
+#23 = IFCRELAGGREGATES('1SfvBitqn5vv1js13bQycY', #1, 'Project Container', 'Project to Spatial Element', #7, (#18));
+#24 = IFCCLASSIFICATION('Default Source', 'Default Edition', '2000-01-01', 'Default Name', $, $, $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('3XlHkb_JX3RBSwRgdO28$y', #1, $, $, (#7), #24);
+#26 = IFCGROUP('12aW0JZoP5v8qiYDw1lDKu', #1, 'A group', $, $);
+#27 = IFCRELDECLARES('0QhXIT61H0CxB10IWPGV9e', #1, 'Project Declaration Relationship', $, #7, (#26));
+#28 = IFCRIGIDOPERATION(#13, #29, IFCPLANEANGLEMEASURE(0.), IFCPLANEANGLEMEASURE(0.), 0.);
+#29 = IFCGEOGRAPHICCRS('EPSG:3065', $, $, $, $, $);
```
Comment (pjanck) [inline files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_B.ifc:36]: `'EPSG:3065'` is not a geographic CRS, but a projected CRS ([source](https://epsg.io/3065)). Please double-check.

**Review by pjanck — CHANGES_REQUESTED** [review]
The data of georeferencing concepts should be revised.

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc (hunk 1 above)
Comment (michelangelo-acca) [reply files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc:5]: Will try to automatize this for next sprint

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc (hunk 3 above)
Comment (michelangelo-acca) [reply files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc:35]: Fixed

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_B.ifc (hunk 4 above)
Comment (michelangelo-acca) [reply files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_B.ifc:36]: Fixed

File: files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc (hunk 2 above)
Comment (michelangelo-acca) [reply files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc:36]: Fixed

**Comment by michelangelo-acca** [comment]
> The data of georeferencing concepts should be revised.

Thanks for the detailed report, everything should be fixed now!

**Review by pjanck — APPROVED** [review]
Thank you for updated files. These seem correct now.

Just out of curiosity: in the B example, you reference [IGM95 with EPSG 4670](https://epsg.io/4670). The documentation states, that it got replaced by EPSG 6706:

> Remarks: Replaced by RDN2008 (CRS code 6706) from 2011-11-10. 

Are these two equal / the latter updates the former?

**Comment by michelangelo-acca** [comment]
> Thank you for updated files. These seem correct now.
> 
> Just out of curiosity: in the B example, you reference [IGM95 with EPSG 4670](https://epsg.io/4670). The documentation states, that it got replaced by EPSG 6706:
> 
> > Remarks: Replaced by RDN2008 (CRS code 6706) from 2011-11-10.
> 
> Are these two equal / the latter updates the former?

Correct, the 6706 replaces the 4670 but in practice IGM95 EPSG 4670 is still the most common used in Italy.

**Review by larswik — APPROVED** [review]

**Pull request merged**